.eas/build/development-build-android.yml

```diff
@@ -7,8 +7,33 @@ build:
 
     - eas/prebuild
 
-    - eas/inject_android_credentials
-
     - eas/run_gradle
 
     - eas/find_and_upload_build_artifacts
+
+    - eas/install_maestro:
+            inputs:
+              maestro_version: 1.35.0
+
+    - eas/start_android_emulator
+
+    - run:
+        name: Install app to running simulator
+        command: |
+          for APP_PATH in android/app/build/outputs/**/*.{apk,aab}; do
+            adb install $APP_PATH
+          done
```

Comment:
It's good for now, but maybe the find and upload artifacts function should return paths to artifacts as output?

Reply:
I wish, yes. I think ideally, the `run_gradle` would know what it built and where, so I can do:

```
- run_gradle:
    id: play_store_build
- run_gradle:
    id: emulator_build
- run
    command: adb install ${ emulator_build.outputs.path }
```
or something like that.